magento/magento2
# Sorting galery media by position

### Description
The gallery of configurable products is not showing the video correctly when the video position in the admin is changed.

### Fixed Issues
1. Fixes magento/magento2#26709: Product video doesn't play when it is not placed first in the sort order.

### Manual testing scenarios
1. Create a configurable product with images in the parent product
2. Add images and a video in the gallery of a child product
3. Go to storefront, select the options of the child that has a video
4. Click on the thumb of the video
5. The video should appear in the gallery with a play button

https://www.loom.com/share/29ef24c26ea04a3d8cd2e03465f071e2

### Contribution checklist
